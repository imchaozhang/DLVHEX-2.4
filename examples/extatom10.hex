bee("b").
aaa("a").
res("ab").

blah(C) :- bee(B), aaa(A), res(C), &testConcat[A,B](C).

% enforce guess and check model generator (see nonmon_guess.hex)
seta(u). seta(v). seta(w).
setc(X) :- &testSetMinus[seta,setd](X), seta(X).
setd(Y) :- &testSetMinus[seta,setc](Y), seta(Y).

% leave only one possibility
setc(u). setc(v). setc(w).
