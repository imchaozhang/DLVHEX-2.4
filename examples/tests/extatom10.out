{bee("b"),aaa("a"),res("ab"),blah("ab"), setc(w), setc(v), setc(u), seta(w), seta(v), seta(u)}
